Modules/GLO/src/ITSTPCMatchingTask.cxx

```diff
@@ -42,6 +42,13 @@ void ITSTPCMatchingTask::initialize(o2::framework::InitContext& /*ctx*/)
     }
   }
 
+  if (auto param = mCustomParameters.find("useTrkPID"); param != mCustomParameters.end()) {
+    ILOG(Debug, Devel) << "Custom parameter - useTrkPID (= add plots for tracking PID): " << param->second << ENDM;
+    if (param->second == "true" || param->second == "True" || param->second == "TRUE") {
```

Comment:
`bool stringUtils::decodeBool(const std::string& value)` does the same as this condition.

Reply:
thanks, I implemented it! Also, while looking for `decodeBool` in the QualityControl I found several implementations, maybe this can be unified